src/database/index.js

```diff
@@ -0,0 +1,18 @@
+import dotenv from "dotenv";
+
+dotenv.config();
+
+module.exports = {
+  development: {
+    url: process.env.DATABASE_URL,
+    dialect: "postgres",
+  },
+  test: {
+    url: process.env.DATABASE_URL,
```

Comment:
should be url: process.env.TEST_DATABASE_URL and insert the key=value in the .env file